### ISSUE TITLE
FN40GL / SCAR (Mk 16/17), G1 / F2000 ("Mk 20")

### ISSUE THREAD

**Issue by irvnriir** [CedricPump/suck#60]
SCAR (Mk 16/17) -s require L (5.56) or H (7.62) trigger adapters for the FN40GL/"MK 13 EGLM", so it would be compatible (may work for some M4, HK 416/417, ..). While F2000 ("Mk 20") require it's **own** G1 module/launcher.

**Comment by CedricPump** [comment]
I m not sure if I 100% understand what you wanna tell me but will have a look into it.

**Comment by irvnriir** [comment]
corrections to compatibility

**Comment by CedricPump** [comment]
so you want to tell me that SCAR version could not use EGLM without adapters and therefore should have an own launcher model?
and that F2000 actually uses a CZ G1 launcher instead of EGLM?
for me this pretty much looks like an EGLM version (futuristic of course) 
https://vignette.wikia.nocookie.net/armedassault/images/7/7c/Arma3-render-mk20eglmcamo.png/revision/latest/scale-to-width-down/300?cb=20191020091311
Or do you mean the FN GL1 not beeing an FN40GL?

**Comment by irvnriir** [comment]
@CedricPump **GL1** (sor, not CZ 805 G1), FN40GL/EGLM is developed from it, with attachment system changed to RIS.
1 or 2 EGLM-s -- idk, adapter should be <100g, and the availability question.

**Comment by CedricPump** [comment]
yeah thanks for the hint. Probably I wont do it right now just because I am too lazy to make another item, but I ll keep it in mind.
